application-xpoll-test/application-xpoll-test-pageobjects/src/main/java/org/xwiki/xpoll/test/po/InPreparationStatusViewPage.java

```diff
@@ -44,9 +49,12 @@ public String getPollStatus()
         return pollStatus.getText();
     }
 
-    public String getPollProposals()
+    public void getPollProposals()
```

Comment:
If the method if called `getPollProposals()` it should return Poll proposals, right?

Reply:
Before it was String. I would have preferred that you've reverted the commit you did before to the page objects.

Reply:
public void getProposals()
    {
        for (WebElement proposal : proposals) {
            pollProposals.add(proposal.getText().trim());
        }
    }

   The other tests have the same pattern and that's why I made it "void" having the name "getPollProposals".
   I could change the function name in these tests.